spec/features/trainer_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+describe "training without cards" do
+  before :all do
+    Card.destroy_all
+    Card.create(original_text: 'дом', translated_text: 'house')
+  end
+
+  it 'no cards' do
+    visit root_path
+    expect(page).to have_content 'Ожидайте наступления даты пересмотра.'
+  end
+end
+
+describe "training with some cards" do
+  before :all do
+    Card.destroy_all
+    card = Card.create(original_text: 'дом', translated_text: 'house')
+    card.update_attribute(:review_date, Time.now)
+    card = Card.create(original_text: 'дом', translated_text: 'house')
+    card.update_attribute(:review_date, Time.now)
+  end
+
+  it 'first visit' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.